test/system/252-quadlet.bats

```diff
@@ -1013,4 +1013,100 @@ EOF
     run_podman rmi $(pause_image)
 }
 
+@test "quadlet - image files" {
+    local image_for_test=$PODMAN_NONLOCAL_IMAGE_FQN
```

Comment:
We could instead use `start_registry` and push an image to the local registry.

Reply:
Great idea. I've adjusted the test code to use a local registry. @vrothberg @edsantiago PTAL